docs/installation/environment-variables.md

```diff
@@ -142,87 +142,10 @@ Configure the following environment variable to enable MicroPowerManager to send
 Certain MicroPowerManager plugins require configuration before they can be used.
 Find below a reference of configurations which are required if the corresponding plugin is used.
 
-#### Airtel
-
-For detailed information see [Airtel Developer Documentation](https://developers.airtel.africa/developer).
-
-| Environment Variable | Default                            | Description                                 |
-| -------------------- | ---------------------------------- | ------------------------------------------- |
-| `AIRTEL_REQUEST_URL` | **Required** (when plugin is used) | The Airtel service URL.                     |
-| `AIRTEL_USERNAME`    | **Required** (when plugin is used) | The Airtel username.                        |
-| `AIRTEL_PASSWORD`    | **Required** (when plugin is used) | The Airtel password.                        |
-| `AIRTEL_IPS`         | `[]`                               | List of IP whitelisted for Airtel services. |
```

Comment:
These seem used.

- https://github.com/EnAccess/micropowermanager/blob/main/src/backend/config/services.php#L35-L40
- https://github.com/EnAccess/micropowermanager/blob/main/src/backend/app/modules/Transaction/Provider/AirtelTransactionProvider.php#L46-L56

please don't remove them

Reply:
AirtelTransactionProvider should be removed. We need to develop a plugin for Airtel If we're planning that in future.